ruby_on_rails/app_initialisation.md

```diff
@@ -15,18 +15,19 @@ You can do this with `gem update rails`. Beware of beta versions.
 
 * Start a new Rails project using `rails new [project-name] --database=postgresql --no-skip-test --skip_action_mailbox` where the `project-name` is exactly the one you chose before.
 
-> ⚠️ You may want to choose a different database than Postgres, but most of the time this will be your choice.
-> If you do not need a DB you may rethink the fact that you may not need Rails at all :) Take a look at [Sinatra](http://www.sinatrarb.com/) or [Angular](https://angular.io/)
+> ⚠️ You may want to choose a different database than Postgres, but most of the time this will be your choice.\
+> If you do not need a DB you may rethink the fact that you may not need Rails at all: Take a look at [Sinatra](http://www.sinatrarb.com/) or [Angular](https://angular.io/)\
 > You might also need actionmailbox of course, so always double-check the parameters that you are using.
-> Check if you've got a `.ruby-version` file. If not, create one and enter the Ruby version.
 
-> ⭐️ This setup does not include either js-bundling nor css-bundling by default. It will start with the simplest possible Rails setup and will use sprockets and importmaps.
-> If you need to do fancy stuff, discuss with your team the opportunity of including a js-bundling and css-bundling tool.
+> ⭐️ This setup does not include either js-bundling nor css-bundling by default.\
+> It will start with the simplest possible Rails setup and will use sprockets and importmaps.\
```

Comment:
```suggestion
> It will start with the simplest possible Rails setup and will use sprockets and importmaps.\
```

can we go for `propshaft`?

Reply:
sprockets is not the simplest possible Rails setup

Reply:
not yet. I don't have a solution to compile scss yet. I have some other moving pieces that I have to figure out. propshaft will be default in rails8

Reply:
@coorasse what about `cssbundling-rails`? why don't you want to got with `propshaft` yet?